### PR TITLE
2023: uefi-sb-with-yocto: add cfp about UEFI Secure Boot and Yocto

### PR DESCRIPTION
**Pull request by TomaszAIR**
CFP proposition for Yocto Summit 2023.11

### CODE REVIEW

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Title
+
+Dealing with UEFI Secure Boot support using Yocto Project
+
+# Abstract
+
+UEFI Secure Boot, a standard within the UEFI framework, enhances embedded
+platform security by verifying trusted keys for loaded images in the UEFI BIOS.
```
Comment (pietrushnic) [inline 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md:8]: @TomaszAIR, that sounds very weird. There is no key verification but signature verification. There can also be other verifications depending on the configuration. Please at least skim through: https://gitlab.com/opensecuritytraining/arch4221_uefi_secure_boot_slides_and_subtitles/-/merge_requests/5/diffs

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+However, integrating it into a project involves challenges such as ensuring UEFI
```
Comment (pietrushnic) [inline 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md:9]: (...) ensuring the level of UEFI specification compliance that UEFI BIOS implementation on a given hardware target provides. Not knowing what is supported may lead to poor choices during integration, and not all UEFI BIOS implementations comply similarly with the specification. Those can also comply with various versions of the specification, which may lead to differences in what and how UEFI Secure Boot should be supported.

@TomaszAIR This is a very long rant, and you should not copy that, but I hope you get my point

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+compliance, planning for certificate rotation to maintain security, and
+seamlessly integrating CI/CD pipelines for component signing. In the upcoming
+discussion, I will explore strategies and solutions for resolving these
```
Comment (pietrushnic) [inline 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md:12]:
```suggestion
presentation, I will explore strategies and solutions for resolving these
```

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md (hunk 1 above)
Comment (TomaszAIR) [reply 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md]: @pietrushnic fixed https://github.com/3mdeb/conferences/pull/9/commits/7b3b6e2c2fdf6915f719670e1cbbd6578c3b4685 

I got `404 Not found` for the provided link.

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md (hunk 2 above)
Comment (TomaszAIR) [reply 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md]: @pietrushnic I just added the first part of that https://github.com/3mdeb/conferences/pull/9/commits/396893ac0961f582b199b120334541410d29657b

Definitely I will talk more about it in the presentation.

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md (hunk 3 above)
Comment (TomaszAIR) [reply 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md]: @pietrushnic fixed https://github.com/3mdeb/conferences/pull/9/commits/1b73d56dfb6bfc4602f347dc954a7f204f2eceb6

File: 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md (hunk 1 above)
Comment (pietrushnic) [reply 2023/YoctoSummit-2023.11/uefi-sb-with-yocto.md]: @TomaszAIR Sorry, I forgot that slides are private. Please check internal repos for the presentation.

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**